cms/templates/js/mock/mock-settings-page.underscore

```diff
@@ -62,6 +62,21 @@
         <span class="tip tip-stacked timezone">(UTC)</span>
       </div>
     </li>
+    <li>
+        <label id="heading-entrance-exam">${_("Entrance Exam")}</label>
```

Comment:
the ```<label> ``` element requires a ```for``` attribute which makes it inappropriate for grouping and labeling the inputs that follow.  I think the text "Entrance Exam" would be more useful as an ```<h3>``` here.